cdap-app-templates/cdap-etl/hydrator-test/src/main/java/co/cask/cdap/etl/mock/spark/streaming/MockSink.java

```diff
@@ -103,15 +96,20 @@ private static PluginClass getPluginClass() {
   }
 
   /**
-   * Get the value associated with the key.
-   * @param key key for which value to be determined
+   * Get the values associated with the specified keys.
+   * @param keys keys for which value to be determined
    * @param tableManager manager for the table
-   * @return the value
+   * @return the values associated with the provided keys
    * @throws Exception
    */
-  public static String getValue(String key, DataSetManager<KeyValueTable> tableManager) throws Exception {
+  public static Set<String> getValues(Set<String> keys, DataSetManager<KeyValueTable> tableManager) throws Exception {
```

Comment:
this is a bit of an odd method, since there is no way to associate which value came from which key. Think it's better to return a Map